canonical-web-and-design/jaas-dashboard
# Fix 404 pages

## Done

- Fix 404 pages

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/fdlkfjdkjf
- Verify 404 page is shown
- Click all other routes and ensure they load as expected

## Details

Fixes #745
